lib/psu_redundancy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+case "$model" in
+    iDRAC[789])
+        $racadm set System.Power.Hotspare.Enable 0
```

Comment:
I am not sure this will work with the latest firmwares. I've seen iDRACs demanding that the new style XML configuration is used. Also the functionality is duplicated into e66243fc7027d5dde2c0648e659bf1a4994b740d , we should keep just one.